llvm/unittests/ADT/STLExtrasTest.cpp

```diff
@@ -1043,6 +1040,14 @@ TEST(STLExtrasTest, getSingleElement) {
   some_namespace::some_struct S;
   S.data = V2;
   EXPECT_EQ(getSingleElement(S), 8);
+
+  // Make sure that we crash on empty or too many elements.
+  SmallVector<int> V4;
+  EXPECT_DEATH(getSingleElement(V4), "expected container with single element");
+  SmallVector<int> V5{12, 13, 14};
+  EXPECT_DEATH(getSingleElement(V5), "expected container with single element");
+  std::list<int> L2;
+  EXPECT_DEATH(getSingleElement(L2), "expected container with single element");
```

Comment:
I think we need to put `#ifndef NDEBUG` around these

Reply:
I'm seeing a mixture of
```c++
#if defined(GTEST_HAS_DEATH_TEST) && !defined(NDEBUG)
...
#endif
```

and

```c++
#if LLVM_ENABLE_ABI_BREAKING_CHECKS
#ifndef NDEBUG
...
#endif
#endif
```

Let's see if the CI is happy with the former...

Reply:
I think the latter is only used for structs with extra data members -- here it's just a simpler helper function so it shouldn't apply